web/src/components/base/ToggleButton.tsx

```diff
@@ -20,6 +20,7 @@ const StyledButton = styled(MuiToggleButton)`
   height: 100px;
   padding: 8px;
   text-align: center;
+  border-color: ${props => props.theme.colors.textDisabledColor} !important;
```

Comment:
🔧 I found another solution both for the border color as well as for the border radius without both setting a custom color and using `!important`:
Add the following in `FeedbackButtons.tsx` to the styling of `StyledToggleButtonGroup` and in `PoiFilters.tsx` to `TileRow`:
``` tsx
  .${toggleButtonGroupClasses.grouped} {
    border-radius: 18px;
  }
  
  .${toggleButtonGroupClasses.middleButton}, .${toggleButtonGroupClasses.lastButton} {
    border-left: 1px solid ${props => (props.theme.vars || props.theme).palette.divider};
  }
```

We could also create a separate file `SpacedToggleButtonGroup` to avoid duplicating the style.

Inspiration taken from https://mui.com/material-ui/react-toggle-button/#spacing.